app/_common/components/Icon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+interface IconProps {
+  id: string;
+  width?: number | string;
+  height?: number;
+}
+
+function Icon(props: React.SVGProps<SVGSVGElement> & IconProps) {
+  const { id, width = "100%", height = "100%", ...rest } = props;
+  return (
+    <svg width={width} height={height} {...rest}>
+      <use href={`#${id}`} />
+    </svg>
```

Comment:
```suggestion
  const { id, size="100%", ...rest } = props;
  return (
    <svg width={size} height={size} {...rest}>
      <use href={`#${id}`} />
    </svg>
```
아이콘 넓이와 높이는 같은 크기로만 만들거라고 생각이 들어서 이렇게 제안드려요.
이렇게 하면 두 개 따로 Props를 전달하지 않고, 한 개의 props로 같은 사이즈 동일하게 넣을 수 있어요!

Reply:
혹시 vercel에 권한이 없어서, 어떤 오류가 발생하는지 확인을 못하고 있어요 ㅠㅠ. 확인 해주실 수 있나요?

Reply:
`npm build` 했을때도 나올거에요!
확인해보니까 엄청 여러 페이지에서 에러가 나타나고 있네요 ㄷㄷ
![스크린샷 2024-03-15 오전 9 55 03](https://github.com/Open-Eye-Im-Developer/MoGakGo-FE/assets/114740795/aa15db23-7076-43f9-aa5a-e6f230746b57)

Reply:
페이지는 수정한 적이 없는데, 제 코드에서 발생한 문제가 아닌 것 같은데 확인해볼게요.

Reply:
어 SVGProvider 때문에 발생하는 문제네요.. 수정해볼게요

Reply:
수정반영했습니다~